ui/src/components/admin/sermonNotes/SermonNotesEditorModal.js

```diff
@@ -0,0 +1,384 @@
+          sermonLink,
+          serviceType,
+          passage,
+        });
+        if (status === 200) {
+          toast({
+            title: 'Announcement Updated',
+            description: 'Your announcement has been updated.',
+            status: 'success',
+            duration: 5000,
+            isClosable: true,
+          });
+        }
+        setIsEditorOpen(false);
+      } else {
+        // Create Unique Sermon ID
+        const formattedData = formatDate(date);
+        const sermonId = `${sermonIdMap[serviceType]}-${formattedData}-${
+          numberOfSermons + 1
+        }`;
+        setSermonId(sermonId);
+        setValue('sermonId', sermonId);
+
+        const { status } = await axios.post('/api/sermon-notes-parent/create', {
+          sermonId,
+          title,
+          subtitle,
+          speaker,
+          sermonSeries,
+          date,
+          imageLink,
+          originalContent,
+          sermonLink,
+          serviceType,
+          passage,
+          isPublished: true,
+        });
+        if (status === 200) {
+          toast({
+            title: 'Announcement Created',
+            description: 'Your announcement has been created.',
+            status: 'success',
+            duration: 5000,
+            isClosable: true,
+          });
+        }
+        setIsEditorOpen(false);
+      }
+    } catch (err) {
+      console.log('Error');
+      toast({
+        title: 'Error',
+        description: 'An error occurred. Please try again later.',
+        status: 'error',
+        duration: 5000,
+        isClosable: true,
+      });
+    }
+  };
+
+  const getData = async () => {
+    const result = await fetchSermonNotes();
+    const numberOfSermonNotes = result.filter((sermonNote) => {
+      return sermonNote.date === date;
+    });
+    if (numberOfSermonNotes.length > 0) {
+      setNumberOfSermons(numberOfSermonNotes.length);
+    } else {
+      setNumberOfSermons(0);
+    }
+  };
+
+  useEffect(() => {
+    getData();
+  }, [date]);
+
+  useEffect(() => {
+    if (editSermonNotesData !== null) {
+      setSermonNotesData(editSermonNotesData);
+    }
+  }, [editSermonNotesData]);
+
+  return (
+    <>
+      <Container
+        borderColor="#FOF357"
+        borderWidth={1}
+        maxW="container.xl"
+        borderRadius={10}
+        p={6}
+      >
+        <Box>
+          <form onSubmit={handleSubmit(onSubmit)}>
+            <Stack gap={6}>
+              <FormControl isRequired>
+                <FormLabel color="#656565" fontWeight="bold">
+                  Title
+                </FormLabel>
+                <Input
+                  id="title"
+                  {...register('title', {
+                    required: 'Title is required',
+                  })}
+                  onChange={(e) => setTitle(e.target.value)}
+                />
+                <FormHelperText color="red">Text is required</FormHelperText>
+              </FormControl>
```

Comment:
try looking into FormErrorLabel and isInvalid props from chakra ui form control, so you will only show the error once its touched by the user and dissapear when the user input smth